### PR TITLE
adding page for storage, fixing broken links and showing hidden prerequisites

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
# What changes were proposed in this pull request?

adding storage requirements page that it was present in https://developer.lightbend.com/docs/cloudflow/current/install/storage.html

Fix the broken link that aimed to it but should aim to prerequisites

Showing the page that @olofwalker created as prerequisites for Kafka and storage

Fix indentation that was missing in the creation of the AKS storage (it was throwing an error when `apply`)

### Why are the changes needed?
to fix the link

### Does this PR introduce any user-facing change?
yes, the doc

### How was this patch tested?
running `make local-preview`

### CODE REVIEW

**Comment by franciscolopezsancho** [comment]
Wait please, don't merge there was another broken link and there's enterprise installation missing on the tree in the chapter administration

**Comment by franciscolopezsancho** [comment]
Now is ready to review.

File: docs/docs-source/docs/modules/administration/pages/installation-prerequisites.adoc [hunk 1]
```diff
@@ -74,6 +74,12 @@ standard (default)   kubernetes.io/gce-pd   2m57s
 
 NOTE: When installing Cloudflow, we will use the name `nfs` to indicate that Cloudflow should use the NFS storage class.
 
+NOTE:: The NFS storage class documented on is very portable and has been verified to work on GKE, EKS, AKS and Openshift.
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/administration/pages/installation-prerequisites.adoc:77]:
```suggestion
NOTE:: The documented NFS storage class is very portable and has been verified to work on GKE, EKS, AKS and Openshift.
```

File: docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc [hunk 2]
```diff
@@ -56,4 +56,4 @@ This opens the Lightbend Console in your browser.
 
 == Upgrading Cloudflow
 
-If you need to upgrade Cloudflow to a newer version, xref:upgrading-cloudflow.adoc[upgrading Cloudflow] in the administration section will show you how.
+If you need to upgrade Cloudflow to a newer version, xref:upgrading-cloudflow.adoc[] in the administration section will show you how.
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc:59]: Will this automatically take the title of the referenced doc?

**Review by RayRoestenburg — APPROVED** [review]

**Pull request merged**